### DISCUSSION TITLE
Ability to use `Toast` imperatively?

### DISCUSSION THREAD

**Discussion by simonschwitzky** [gluestack/gluestack-ui#480] [Q&A]
NativeBase provided functionality to show Toasts outside of React components, does Gluestack offer a similar function? (Docs don't mention this)

**Comment by makkarMeenu** [comment]
Hi @saaymeen , thankyou for requesting that.
We will work on the API for standalone toast that works outside React component just like NativeBase [Toast](https://docs.nativebase.io/toast#h3-standalone-toast).
